one_fm/fixtures/assignment_rule.json

```diff
@@ -1,132 +1,4 @@
 [
- {
-  "assign_condition": "workflow_state == 'Pending Approver'",
-  "assignment_days": [
-   {
-    "day": "Monday",
-    "parent": "Shift Request Pending Approval",
-    "parentfield": "assignment_days",
-    "parenttype": "Assignment Rule"
-   },
-   {
-    "day": "Tuesday",
-    "parent": "Shift Request Pending Approval",
-    "parentfield": "assignment_days",
-    "parenttype": "Assignment Rule"
-   },
-   {
-    "day": "Wednesday",
-    "parent": "Shift Request Pending Approval",
-    "parentfield": "assignment_days",
-    "parenttype": "Assignment Rule"
-   },
-   {
-    "day": "Thursday",
-    "parent": "Shift Request Pending Approval",
-    "parentfield": "assignment_days",
-    "parenttype": "Assignment Rule"
-   },
-   {
-    "day": "Friday",
-    "parent": "Shift Request Pending Approval",
-    "parentfield": "assignment_days",
-    "parenttype": "Assignment Rule"
-   },
-   {
-    "day": "Saturday",
-    "parent": "Shift Request Pending Approval",
-    "parentfield": "assignment_days",
-    "parenttype": "Assignment Rule"
-   },
-   {
-    "day": "Sunday",
-    "parent": "Shift Request Pending Approval",
-    "parentfield": "assignment_days",
-    "parenttype": "Assignment Rule"
-   }
-  ],
-  "close_condition": "workflow_state in (\"Approved\", \"Rejected\", \"Draft\")",
-  "custom_routine_task": null,
-  "description": "Automatic Assignment",
-  "disabled": 0,
-  "docstatus": 0,
-  "doctype": "Assignment Rule",
-  "document_type": "Shift Request",
-  "due_date_based_on": null,
-  "field": "shift_approver",
-  "is_assignment_rule_with_workflow": 0,
-  "last_user": "abdullah@one-fm.com",
-  "modified": "2024-07-22 13:32:41.603305",
-  "name": "Shift Request Pending Approval",
-  "priority": 0,
-  "rule": "Based on Field",
-  "unassign_condition": "workflow_state in (\"Approved\", \"Rejected\", \"Draft\")",
```

Comment:
Why we removed this Assignment Rule

Reply:
I didn't remove it. I created new assignment rule based on the requirement and I bench export-features to have the changes in my code.